### PR TITLE
Added examples for the Sanitizer parameters API

### PR DESCRIPTION
**Pull request by ghost**
This is related to the https://github.com/mdn/content/pull/18790#event-7061644512
I added examples for the Sanitizer parameters API. 
I appreciate feedbacks.

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/sanitizer/sanitizer/index.md [hunk 1]
```diff
@@ -73,12 +73,49 @@ This example shows the result of sanitizing a string with disallowed `script` el
...
 const unsanitized = "abc <script>alert(1)<" + "/script> def";
-const sanitized =  new Sanitizer().sanitizeFor("div", unsanitized);
+const sanitized = new Sanitizer().sanitizeFor("div", unsanitized);
 // Result (innerHTML of 'sanitized'): script will be removed: "abc alert(1) def"
 ```
 
 <!-- Add other examples showing use of parameter -->
 
+```js
+const santizerElementSample = to_node("<div style="padding: 10px;" class="container" id="wrapper"><img  style="display: block; box-sizing: border-box;" class="img-hero" src="./picture.jpg" alt=""></div>);
```
Comment (Josh-Cena) [inline files/en-us/web/api/sanitizer/sanitizer/index.md:83]: This is invalid syntax. Do you mean

```suggestion
const santizerElementSample = to_node('<div style="padding: 10px;" class="container" id="wrapper"><img  style="display: block; box-sizing: border-box;" class="img-hero" src="./picture.jpg" alt=""></div>');
```

Also, I'm not sure `to_node` is introduced anywhere.

Comment (najmiehsa) [reply files/en-us/web/api/sanitizer/sanitizer/index.md]: Good point. according to [documentation](https://wicg.github.io/sanitizer-api/#attribute-drop-list) "we’ll need a way to create DOM nodes" but I am missing  this part ``const to_node = str => document.createRange().createContextualFragment(str);``

How can I add this without making it confusing?

**Comment by jpmedley** [comment]
@najmiehsa, Thank you for submitting these examples. Unfortunately, when this lands in Chrome beta next week, only `Element.setHTML()` will be shipping. One of the spec authors, who is also Chrome's lead on the project tells me that the authors generally want to change the design. In other words, in Chrome's release, you'll be able to pass `new Sanitizer()` as the second parameter to `setHTML()`, but you will not be able to call `sanitize()` and `sanitizeFor()`. 

Given that the Firefox implementation of these methods is still behind a flag, I do not think we should merge this pull request. 

cc: @Rumyra

**Comment by teoli2003** [comment]
Yes, we need one implementation (not behind a flag) before merging a feature into MDN. We document browsers and not specs.

**Comment by hamishwillee** [comment]
I'm going to close this because the time taken for this to be "supported" is still open ended (and as Joe says, might not be quite this implementation). Thanks @najmiehsa for doing this - sorry to ask you for it with this result.

@jpmedley Are you taking care of the browser compat updates required? If not, I would need more information to update this.

What I'm seeing now:

- [`Element.setHTML()](https://developer.mozilla.org/en-US/docs/Web/API/Element/setHTML) not marked as supported in preview.
- `sanitize()` and `sanitizeFor()` are rendering as "preview browser support". The [BCD shows them](https://github.com/mdn/browser-compat-data/blob/main/api/Sanitizer.json#L255) as supported in 105. 
 
Anyway, it is not clear to me from the current BCD whether they would be callable in the preview version. My assumption as it is now is that they would be.

**Comment by hamishwillee** [comment]
@teoli2003 PS I merged the original docs for this as part of an FF release project. Did not appreciate at that time that a non-pref implementation was required.

**Pull request closed without merging**